bin/filter-json

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+// Read JSON from stdin, keep keys passed as arguments, write the result to stdout as JSON
+
+let args = process.argv.slice(2);
+let data = [];
+
+process.stdin.on("data", chunk => {
+	data.push(chunk);
+});
+
+process.stdin.on("end", () => {
+	let obj = JSON.parse(Buffer.concat(data));
+	Object.keys(obj).
+		filter(key => !args.includes(key)).
+		forEach(key => delete obj[key]);
+	console.log(JSON.stringify(obj));
```

Comment:
Should we be consistent in whether we use `console.*` or `process.stdout.write` to generate output?